packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides

```diff
@@ -1020,6 +1019,34 @@
     transform: translate(-50%, -50%);
 }
 
+/* SKIN | Link */
+.ui.button.link {
+    background: none !important;
+    color: var(--primary-600);
+    font-weight: 400;
+}
+
+/* SKIN | Link | STATE | Active */
+.ui.button.link:focus,
+.ui.button.link:hover,
+.ui.button.link:active {
+    text-decoration: underline;
```

Comment:
In the best world we would not use      text-decoration: underline;  but a border bottom